architecture/effects/README.md

```diff
@@ -85,4 +85,4 @@ Now it is totally okay if this does not really make sense yet! That only really
 >   - Avoid setup/teardown phases when testing `update` logic.
 >   - Cache and batch effects, minimizing HTTP connections or other resources.
 >
-> So without going too crazy on details, pretty much all the nice guarantees and tools you have in Elm come from the choice to treat effects as data! I think this will make more sense as you get deeper into Elm.
\ No newline at end of file
+> So without going too crazy on details, pretty much all the nice guarantees and tools you have in Elm come from the choice to treat effects as data! I think this will make more sense as you get deeper into Elm.
```

Comment:
Hmm, I made my edits in GitHub's web editor, and I'm pretty sure I didn't edit this line.